diff --git a/node_modules/@apphosting/adapter-nextjs/dist/utils.js b/node_modules/@apphosting/adapter-nextjs/dist/utils.js
index 2352c8b..178af16 100644
--- a/node_modules/@apphosting/adapter-nextjs/dist/utils.js
+++ b/node_modules/@apphosting/adapter-nextjs/dist/utils.js
@@ -15,7 +15,7 @@
     const { version } = JSON.parse(fs_1.default.readFileSync(path_1.default.join(cwd, 'package.json'), 'utf-8'));
     const isVulnerable = semver_1.default.satisfies(version, '>=15.0.0-rc.0 <15.3.6', {
         includePrerelease: true,
     });
-    if (isVulnerable) {
-        throw new Error(`CVE-2025-55182: Vulnerable Next version ${version} detected. Deployment blocked. Update your app's dependencies to a patched Next.js version and redeploy: https://nextjs.org/blog/CVE-2025-66478#fixed-versions`);
-    }
+    // if (isVulnerable) {
+    //     throw new Error(`CVE-2025-55182: Vulnerable Next version ${version} detected. Deployment blocked. Update your app's dependencies to a patched Next.js version and redeploy: https://nextjs.org/blog/CVE-2025-66478#fixed-versions`);
+    // }
 }
 exports.checkNextJSVersion = checkNextJSVersion;
